lib/plugins/create/templates/README.md

```diff
@@ -0,0 +1,29 @@
+# Templates
+
+## Java
+
+### Maven - Quick Start
+
+Follow these simple steps to install the alpha, create and deploy your java service using maven, run your function and remove the service afterwards.
```

Comment:
I'm not sure if we should specifically go into the alpha here, or if we simply focus on this specific template. Otherwise we have to go over it once we've moved to Beta or prod. Imho we can simply say that the serverless tool and Maven need to be installed beforehand and can link to the getting started guide on that on the main docs page